scss/_patterns_card.scss

```diff
@@ -1,62 +1,107 @@
 @import 'settings';
 
+@mixin vf-p-card {
+  %p-card {
+    @extend %bg--x-light;
+    margin-bottom: $spv-inter--scaleable;
+    overflow: auto; // prevent overflow of child margins
+    padding: $spv-intra--scaleable;
+  }
+
+  @include vf-p-card-default;
+  @include vf-p-card-highlighted;
+  @include vf-p-card-overlay;
+  @include vf-p-card-muted;
+  @include vf-p-card-specific-content;
+}
+
 @mixin vf-p-card-default {
   .p-card {
+    @extend %bg--x-light;
+    @extend %is-bordered;
+    @extend %has-round-corners;
     @extend %p-card;
-    border: 1px solid $color-mid-light;
-    padding: $spv-intra--scaleable - $px $spv-intra--scaleable $spv-intra--scaleable;
+    padding: $spv-intra--scaleable - $px;
```

Comment:
Whats the difference between this line and `padding: $spv-intra--scaleable;`

Reply:
borders are factored into the padding calculation to keep overall size of the item the same. So here I'm just subtracting 1 px. To keep everything symmetric, it is done not only vertically, but on the sides too.

Reply:
Ah cool, thanks